recipes/sqlalchemy-access/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "sqlalchemy-access" %}
+{% set version = "2.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sqlalchemy-access-{{ version }}.tar.gz
+  sha256: 61b577dddc7975f64a567a711105450f56a1d5057e7f0ef1c5772b62a7bc2b1c
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - sqlalchemy >=2.0.0
+    - pyodbc >=4.0.27
+    - pywin32-on-windows
```

If this package is `win` only, you could add the following
```suggestion
    - pywin32-on-windows
    - __win
```

---

Why didn't you end up using this?

---

Hi @xhochy,
I tried it, but the check failed. So then I tried a lot of other things, and failed, and failed, and failed. Until it did not (allowing only for installation on windows, since the `sqlalchemy-access` plugin is for reading Microsoft Access and it is only sensible to do on windows).
Afterthought: I probably did something wrong when merging your suggested change, and it might also have worked. 
Question: what does the `__win` do?

---

> I tried it, but the check failed.

It failed with which result? In some cases a failing CI on other platforms is OK.

`__win` adds a dependency on Windows as the runtime environment, i.e. the resulting package, even if marked as `noarch`, is only installable on Windows

---

I see. It failed only on the non-windows environments. I will try again with your version.